### DISCUSSION TITLE
Reducing to 2 dimensions

### DISCUSSION THREAD

**Discussion by scottthomaswx** [pydata/xarray#5486] [Q&A] [answered]
I am working on a project with a dataset with time, latitude, and longitude as dimensions. I have used .loc to pull the data for a specific time, is there a way I can make a new array with that data with only latitude and longitude as dimensions so I can plot it on a map?

**Comment by max-sixty** [comment, marked as answer]
Please take a look at https://github.com/pydata/xarray/discussions/5404 — thanks
